docs/resources/scalr_variable.md

```diff
@@ -35,7 +35,12 @@ resource "scalr_variable" "example" {
 * `category` - (Required) Indicates if this is a Terraform or environment variable. Allowed values are `terraform` or `env`.
 * `hcl` - (Optional) Set (true/false) to configure the variable as a string of HCL code. Has no effect for `category = "env"` variables. Default `false`.
 * `sensitive` - (Optional) Set (true/false) to configure as sensitive. Sensitive variable values are not visible after being set. Default `false`.
-* `workspace_id` - (Required) The workspace that owns the variable, specified as an ID, in the format `ws-<RANDOM STRING>`.
+* `final` - (Optional) Set (true/false) to configure as final. Indicates whether the variable can be overridden on a lower scope down the Scalr organizational model. Default `false`.
```

Comment:
@pgale61 review the documentation please

Reply:
LGTM